packages/@cdklabs/cdk-cicd-wrapper/src/plugins/security/AccessLogsForBucketPlugin.ts

```diff
@@ -0,0 +1,40 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+import { IAspect, Aspects, Names } from 'aws-cdk-lib';
+import { CfnBucket } from 'aws-cdk-lib/aws-s3';
+import { Construct } from 'constructs';
+import { GlobalResources, IPlugin, ResourceContext } from '../../common';
+
+/**
+ * Plugin to enable access logs for an S3 bucket.
+ */
+export class AccessLogsForBucketPlugin implements IPlugin {
+  readonly name: string = 'AccessLogsForBucketPlugin';
+
+  readonly version: string = '1.0';
```

Comment:
why hardcoded version, what does it help?

Reply:
this is for supporting third-party plugins. Those moslikely be versioned and with this we will have a way to provide clean information about the used plugins and versions.

Whenever we will change the Plugin we will bump the version number as well, so the end users will have a clarity that a new plugin version is used. Later on we will be able to support multiple parallel versions, etc.